Makefile

```diff
@@ -15,8 +15,8 @@ clean:
 
 
 latest-alpine:
-	docker build -t $(PROJECT_APLINE):latest buildtool-alpine
+	docker build -t $(PROJECT_APLINE)-3.9:1.12 buildtool-alpine
```

Comment:
I believe it's better to keep the version in a variable

1) to not forgot to chang in multiple places
2) to allow override from comamnd line if it ever necessary.

Maybe even sed Dockerfile (or use Dockerfile.template) to replace version there